spec/spec.md

```diff
@@ -190,59 +199,7 @@ requirement.
 
 ::: example Presentation Definition - Basic Example
 ```json
-{
-  // VP, OIDC, DIDComm, or CHAPI outer wrapper
-  "presentation_definition": {
-    "id": "32f54163-7166-48f1-93d8-ff217bdb0653",
-    "input_descriptors": [
-      {
-        "id": "banking_input",
-        "name": "Bank Account Information",
-        "purpose": "We need your bank and account information.",
-        "schema": [
-          {
-            "uri": "https://bank-standards.com/customer.json"
-          }
-        ],
-        "constraints": {
-          "limit_disclosure": "required",
-          "fields": [
-            {
-              "path": ["$.issuer", "$.vc.issuer", "$.iss"],
-              "purpose": "The claim must be from one of the specified issuers",
-              "filter": {
-                "type": "string",
-                "pattern": "did:example:123|did:example:456"
-              }
-            }
-          ]
-        }
-      },
-      {
-        "id": "citizenship_input",
-        "name": "US Passport",
-        "schema": [
-          {
-            "uri": "hub://did:foo:123/Collections/schema.us.gov/passport.json"
-          }
-        ],
-        "constraints": {
-          "fields": [
-            {
-              "path": ["$.credentialSubject.birth_date", "$.vc.credentialSubject.birth_date", "$.birth_date"],
-              "filter": {
-                "type": "string",
-                "format": "date",
-                "minimum": "1999-5-16"
-              }
-            }
-          ]
-        }
-
-      }
-    ]
-  }
-}
+[[import ../test/presentation-definition/example.json ]]
```

Comment:
I ended up going with `insert`, and the path is assumed to be from the root of the project, so could you change this to : [[insert: ./test/presentation-definition/example.json]]?